### PR TITLE
README: using custom operand image

### PR DESCRIPTION
**Pull request by alebedev87**

### CODE REVIEW

**Comment by alebedev87** [comment]
Problem with bundle instillation, already happened before https://coreos.slack.com/archives/CBN38N3MW/p1646082553297829

/retest

**Comment by alebedev87** [comment]
The custom ingress controller was never reconciled by the ingress operator: no logs about this controller.
/retest

File: README.md [hunk 1]
```diff
@@ -160,8 +160,42 @@ Prepare your environment for the installation commands.
...
+    oc apply -k config/samples/aws
+    ```
+    *Note*: For other providers, see `config/samples/`.
+
+## Using custom operand image
+1. _Optional_: you may need to link the registry secret to the operand's service account if your custom image is not public ([Doc link](https://docs.openshift.com/container-platform/4.10/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets)):
+
+    a. Create a secret with authentication details of your image registry:
+    ```sh
+    oc -n external-dns-operator create secret generic extdns-pull-secret --type=kubernetes.io/dockercfg --from-file=.dockercfg=${XDG_RUNTIME_DIR}/containers/auth.json
```
Comment (DhritiShikhar) [inline README.md:177]: I get the error -> file not found
```
➜  external-dns-operator git:(main) ✗ oc -n external-dns-operator create secret generic extdns-pull-secret --type=kubernetes.io/dockercfg --from-file=.dockercfg=${XDG_RUNTIME_DIR}/containers/auth.json
error: error reading /run/user/1000/containers/auth.json: no such file or directory
```

**Pull request merged**